agent/ecscni/types_windows.go

```diff
@@ -50,11 +52,9 @@ type VPCENIPluginConfig struct {
 	// ENIMACAddress is the MAC address of the eni.
 	ENIMACAddress string `json:"eniMACAddress"`
 	// ENIIPAddress is the is the ipv4 of eni.
-	ENIIPAddress string `json:"eniIPAddress"`
+	ENIIPAddress []string `json:"eniIPAddresses"`
```

Comment:
i think this should be `ENIIPAddresses` and below should be `GatewayIPAddresses`?